pyproject.toml

```diff
@@ -123,3 +123,11 @@ target-version = ['py38']
 [tool.check-wheel-contents]
 # W009: Wheel contains multiple toplevel library entries
 ignore = "W009"
+
+[tool.mypy]
```

Comment:
Currently we configure mypy in `setup.cfg`:

https://github.com/pytest-dev/pytest/blob/d97295730330f7686a155dd6482405ad4a823b1d/setup.cfg#L93-L104

Reply:
Ah didn't spot that